### PR TITLE
Image fixed

### PR DESCRIPTION
**Pull request by jlaura**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,11 +1,11 @@
-FROM python:3-alpine
+FROM continuumio/miniconda
```
Comment (Kelvinrr) [inline Dockerfile]: should this be miniconda3?

**Comment by Kelvinrr** [comment]
running it gives me the error:
`docker: Error response from daemon: OCI runtime create failed: container_linux.go:348: starting container process caused "exec: \"python3\": executable file not found in $PATH": unknown.`

File: Dockerfile (hunk 1 above)
Comment (jlaura) [reply Dockerfile]: Good catch, fixed it.

**Comment by jlaura** [comment]
Success!

```bash
Step 8/10 : EXPOSE 8080
 ---> Running in b4d562476055
Removing intermediate container b4d562476055
 ---> 45db5e30aca0
Step 9/10 : ENTRYPOINT ["python"]
 ---> Running in af4a2b5e4caa
Removing intermediate container af4a2b5e4caa
 ---> 293708ccc5fd
Step 10/10 : CMD ["-m", "pfeffernusse"]
 ---> Running in 265ceb0ed1fa
Removing intermediate container 265ceb0ed1fa
 ---> 4c35f6a8ce9d
Successfully built 4c35f6a8ce9d
Successfully tagged pfeffernusse:latest
 * Serving Flask app "__main__" (lazy loading)
 * Environment: production
   WARNING: Do not use the development server in a production environment.
   Use a production WSGI server instead.
 * Debug mode: off
 * Running on http://0.0.0.0:8080/ (Press CTRL+C to quit)
```

**Comment by Kelvinrr** [comment]
:+1:

**Pull request merged**